### PR TITLE
add initial types for clmtrackr

### PR DESCRIPTION
**Pull request by hellochar**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

--------

Basic incomplete typings for what I use of https://github.com/auduno/clmtrackr, based off of https://www.auduno.com/clmtrackr/docs/reference.html . FYI @auduno

### CODE REVIEW

**Review by plantain-00 — APPROVED** [review]

**Comment by andy-ms** [comment]
Approved by third party. PR appears ready to merge pending review by a maintainer.

File: types/clmtrackr/index.d.ts
Comment (andy-ms) [inline types/clmtrackr/index.d.ts]: When I require the module I just see `tracker` and `version` properties directly, with no `default` property. This is on node.js using:
```ts
//hax to make it not crash on load
global.window = global;
window.Event = class Event {}
console.log(require("clmtrackr"));
```
Could you check if your module loader is injecting a default export where there is none?

File: types/clmtrackr/index.d.ts
Comment (andy-ms) [inline types/clmtrackr/index.d.ts]: Nit: Just use `Position`.

File: types/clmtrackr/index.d.ts
Comment (andy-ms) [inline types/clmtrackr/index.d.ts]: This is only used in one place, inline?

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**
